content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md

```diff
@@ -86,7 +86,7 @@ kubectl config --kubeconfig=config-demo set-credentials experimenter --username=
 ```
 
 {{< note >}}
-To delete a user you can run `kubectl config unset users.<name>`
+To delete a user, you can run `kubectl --kubeconfig=config-demo config unset users.<name>`. To remove a cluster from kubectl, you can run `kubectl --kubeconfig=config-demo config unset clusters.<name>`, and to remove a context, you can run `kubectl --kubeconfig=config-demo config unset contexts.<name>`.
```

Comment:
I think this would render better with forced line breaks after each sentence.

Reply:
Hello @Kartik494 , @sftim 
I think this note would be easier to read if the commands were put in an unordered list.
The command should be emphasized (per style guide and previously suggested). Such as:

```
{{< note >}}

- To delete a user, run `kubectl --kubeconfig=config-demo config unset users.<name>`
- To remove a cluster, run `kubectl --kubeconfig=config-demo config unset clusters.<name>`
- To remove a context, run `kubectl --kubeconfig=config-demo config unset contexts.<name>`

{{< /note >}}
```

Reply:
Hi @sftim @kbhawkey I have addressed the changes as mention.
Thanks